source/StepBro.ExecutionHelper/ExecutionHelperMessages.cs

```diff
@@ -11,7 +11,8 @@ public enum ShortCommand
         None,
         CloseApplication,
         CloseConnection,
-        Acknowledge
+        Acknowledge,
+        Error
```

Comment:
Maybe it should be a separate message, 'CommandError', with an error description string.
The Access functions could directly report the error through the context reference.